portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2991,6 +2989,18 @@ private DLFileEntry _moveFileEntryImpl(
 		return dlFileEntry;
 	}
 
+	private void _notify(long userId, int emailType, DLFileVersion fileVersion)
```

Comment:
OPTIONAL: 

We could have the implementation of this method directly in line 2713.

Reply:
On this PR I just extracted the _notify( method.

it's used 2 times.
Do you think is better to not extract it?

Reply:
You are right , I hadn't noticed the first call.

Reply:
I will discard this.